### ISSUE TITLE
dragAndMove true and setAllowScrolling not working

### ISSUE THREAD

**Issue by reinchang** [alvarotrigo/fullPage.js#2506]
Hi,
I used the extenstion: dragAndMove and need to disable setAllowScrolling when users enter specific page, after they complete some missions (answer the question or something) then enable the allowscrolling (therefore they can go other page).
But it seems not working, are there any suggestions?
Thank you.

**Comment by ushowjack** [comment]
@reinchang Hello，guy.
I just start to use fullpage.js today. Now I have a problem with how to use the the file "jquery.fullpage.extensions.min.js". What's its usage? I want to use the action of dragAndMove, but it don't work. And the codesource of example page has a line of coding 
```JavaScript
	<script type="text/javascript" src="../fullpage.dragAndMove.limited.min.js"></script>
```
But I can't find it anywhere, even google the file.

**Comment by alvarotrigo** [comment]
@reinchang you found a bug :) I've sent you a mail with a new version of the extension that solves it.

**Comment by alvarotrigo** [comment]
@ushowjack as detailed in the documentation, the `dragAndMove` option is provided as a fullPage.js extension you need to purchase [from here](http://alvarotrigo.com/fullPage/extensions/).

**Comment by reinchang** [comment]
@alvarotrigo You're the best. Thank you for the immediately fixed. It works perfectly now. 

p.s. there is another tiny problem(I'm not sure if other users report this problem, or just happen to me )  => when I set $.fn.fullpage.setAllowScrolling(false) or $.fn.fullpage.setAllowScrolling(false,'all')=>the pages still can move, when I set $.fn.fullpage.setAllowScrolling(false,'up,down') =>it works fine.

**Comment by alvarotrigo** [comment]
> p.s. there is another tiny problem(I'm not sure if other users report this problem, or just happen to me ) => when I set $.fn.fullpage.setAllowScrolling(false) or $.fn.fullpage.setAllowScrolling(false,'all')=>the pages still can move, when I set $.fn.fullpage.setAllowScrolling(false,'up,down') =>it works fine.

Thanks for letting me know! I'll check it out, probably I forgot about the global option :)

**Comment by alvarotrigo** [comment]
Solved in the new release 2.9.3
